tests/unit/utils/test_version.py

```diff
@@ -950,3 +953,74 @@ def test_opengl_vendor():
     """Simply call version.opengl_vendor() and see if it doesn't crash."""
     pytest.importorskip("PyQt5.QtOpenGL")
     return version.opengl_vendor()
+
+
+class HTTPPostStub(QObject):
+
+    """A stub class for HTTPClient.
+
+    Attributes:
+        url: the last url send by post()
+        data: the last data send by post()
+    """
+
+    success = pyqtSignal(str)
+    error = pyqtSignal(str)
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.url = None
+        self.data = None
+
+    def post(self, url, data=None):
+        self.url = url
+        self.data = data
+
+
+@pytest.fixture
+def pbclient():
+    http_stub = HTTPPostStub()
+    client = pastebin.PastebinClient(http_stub)
+    return client
+
+
+def test_pastebin_version(pbclient, monkeypatch):
+    """Test version.pastebin_version() twice."""
+    patches = {
+        '_path_info': lambda: {'PATH DESC': 'PATH NAME'},
+        '_uptime': lambda: datetime.timedelta(hours=1, minutes=23, seconds=45),
+    }
+
+    for name, val in patches.items():
+        monkeypatch.setattr('qutebrowser.utils.version.' + name, val)
```

Comment:
I think it'd be better to just patch away `version.version()` as we don't want to test that at all here.

Reply:
d'oh, why didn't I think of that